AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -627,6 +627,8 @@ m.blog.naver.com##.btm_banner
 wap.business-standard.com###popup123
 ko.dict.naver.com##.banner_vkorean_wrap
 nv.ua##.selfpromo_new
+m.hindustanttimes.com##.storyDetail > section[style^="clear: both;padding:0px 11px;"]
+m.hindustanttimes.com##div.mb20[style="text-align:center;"] > a > img
```

Comment:
Why not just `||m.hindustantimes.com/res/img/*/app.png`? And I guess it's for mobile-app section.
I don't see `m.hindustanttimes.com##.storyDetail > section[style^="clear: both;padding:0px 11px;"]`

Reply:
paddin0px element:
![screen](https://user-images.githubusercontent.com/68852737/92941504-1f24e600-f459-11ea-8329-5e7cf0e5d62b.jpg)

Reply:
>And I guess it's for mobile-app section.
Mobile-app or mobile-specific? Looks like self-promo to me though.